CHANGELOG.md

```diff
@@ -2,23 +2,98 @@
 
 ## Unreleased
 
+## v0.105.0
```

Comment:
We had some discussion, we want to have the header message here about what upstream versions are included in this release, something like:
```
This Splunk OpenTelemetry Collector release includes changes...
```
_Refer to previous release changelogs for full content_